ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -1,2 +1,15 @@
 """Includes Mindspore Frontend functions listed in the TODO list
 https://github.com/unifyai/ivy/issues/14951."""
+
+# local
+import ivy
+from ivy.func_wrapper import with_supported_dtypes
+
+
+@with_supported_dtypes({"2.0.0 and below": ("float16", "float32")}, "mindspore")
+def selu(input_x):
+    alpha = 1.67326324
+    scale = 1.05070098
+    ret = ivy.where(input_x > 0, input_x, alpha * ivy.expm1(input_x))
+    arr = scale * ret
+    return ivy.astype(arr, input_x.dtype)
```

Comment:
Was there a reason to not use `ivy.selu` here?

Reply:
First PR mistakes 😅 I hesitated since ivy.selu is in the experimental API. I added the ivy.selu method now and tested locally by creating arrays, it works the same. 

I have a question, when do we move these experimental functions out of the experimental API and when are they considered stable enough? I checked other frontends as well, torch, tensorflow and jax use ivy.selu while paddle has the same longer implementation that I had written here.

Reply:
The ivy function probably got merged recently, after the frontends were implemented. We will have to update the frontend functions to use that. Frontends should be as short as possible. We've got a moderator responsible for tracking the stability of experimental functions and moving them to the main api.